fearless/Modules/StakingPool/StakingPoolInfo/StakingPoolInfoViewLayout.swift

```diff
@@ -22,6 +26,24 @@ final class StakingPoolInfoViewLayout: UIView {
         return view
     }()
 
+    let statusView: GenericTitleValueView<TitleStatusView, IconDetailsView> = {
+        let statusView = TitleStatusView()
+        statusView.mode = .indicatorTile
+        statusView.spacing = 8.0
+        statusView.titleLabel.font = .capsTitle
+
+        let detailsView = IconDetailsView()
+        detailsView.mode = .detailsIcon
+        detailsView.spacing = 0.0
+        detailsView.detailsLabel.font = .capsTitle
+        detailsView.detailsLabel.textColor = R.color.colorTransparentText()
+        detailsView.imageView.image = nil
+        detailsView.iconWidth = 24.0
+        detailsView.detailsLabel.numberOfLines = 1
+
+        return GenericTitleValueView(titleView: statusView, valueView: detailsView)
+    }()
+
     let infoBackground: TriangularedView = {
```

Comment:
most of views should be private. And actually mb we should think about setting targets through handlers in view models, so we can keep more of them private.